Packs/Wiz/Integrations/Wiz/requirements.txt

```diff
@@ -0,0 +1,7 @@
+pylint
```

Comment:
This file is not required. You can discard it.

Reply:
ACK. Removed.

Reply:
917b05ee911e1bce12d1b0f8f43661e89a99b596